hazelcast/src/test/java/com/hazelcast/cache/CacheCloseTest.java

```diff
@@ -1,160 +0,0 @@
-package com.hazelcast.cache;
-
-import com.hazelcast.cache.impl.CacheEventListener;
-import com.hazelcast.cache.impl.HazelcastServerCachingProvider;
-import com.hazelcast.cache.impl.ICacheService;
-import com.hazelcast.cache.impl.client.CacheSingleInvalidationMessage;
-import com.hazelcast.cache.impl.operation.CacheDestroyOperation;
-import com.hazelcast.config.CacheConfig;
-import com.hazelcast.config.Config;
-import com.hazelcast.core.HazelcastInstance;
-import com.hazelcast.instance.HazelcastInstanceProxy;
-import com.hazelcast.spi.impl.NodeEngineImpl;
-import com.hazelcast.spi.impl.operationservice.InternalOperationService;
-import com.hazelcast.test.AssertTask;
-import com.hazelcast.test.HazelcastParallelClassRunner;
-import com.hazelcast.test.TestHazelcastInstanceFactory;
-import com.hazelcast.test.annotation.ParallelTest;
-import com.hazelcast.test.annotation.QuickTest;
-import org.junit.Test;
-import org.junit.experimental.categories.Category;
-import org.junit.runner.RunWith;
-
-import javax.cache.CacheManager;
-import javax.cache.spi.CachingProvider;
-import java.util.concurrent.ExecutionException;
-import java.util.concurrent.atomic.AtomicInteger;
-
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertNotNull;
-import static org.junit.Assert.assertNull;
-import static org.junit.Assert.assertTrue;
-
-@RunWith(HazelcastParallelClassRunner.class)
-@Category({QuickTest.class, ParallelTest.class})
-public class CacheCloseTest
```

Comment:
I was first concerned that we might miss code coverage for the `Cache.close()` method if this test is removed, but it seems that there are some other `testThisAndThat_afterCacheClose`, so we should be fine.